File: app/src/main/java/org/fattili/easypopup_demo/TestPop.java
package org.fattili.easypopup_demo;

import android.app.Activity;
import android.view.View;

import org.fattili.easypopup.BasePopView;

/**
 * 2021/2/22
 *
 * @author dengzhenli
 */
public class TestPop extends BasePopView {
    public TestPop(Activity activity) {
        super(activity);
    }

    @Override
    public void initView(View view) {

    }

    @Override
    public void initData() {

    }

    @Override
    public int getLayoutId() {
        return R.layout.pop_test;
    }
}

File: app/src/main/java/org/fattili/easypopup_demo/kt/TestPop.kt
package org.fattili.easypopup_demo.kt

import android.app.Activity
import android.view.View
import kotlinx.android.synthetic.main.pop_example.view.*
import org.fattili.easypopup.view.base.BasePopView
import org.fattili.easypopup_demo.R

/**
 * 2021/2/22
 *
 * @author dengzhenli
 */
class TestPop(activity: Activity) : BasePopView(activity) {
    override fun initView(view: View?) {
        pop_example_text.text = "我是普通弹出窗"
    }
    override fun initData() {}
    override fun getLayoutId(): Int {
        return R.layout.pop_test
    }

    override fun onPopDismiss() {}
    override fun outClickable(): Boolean {
        return true
    }
}